src/PhpBrew/VariantBuilder.php

```diff
@@ -551,6 +552,26 @@ public function __construct()
             }
         };
 
+        $this->variants['sodium'] = function (Build $build, $val = null) {
```

Comment:
The user-provided prefix probably shouldn't be ignored. Look at:
https://github.com/phpbrew/phpbrew/blob/b2171dc8bc4b4de7cfde39b34719cb337f7454d2/src/PhpBrew/VariantBuilder.php#L418-L420